pennylane/measurements/sample.py

```diff
@@ -230,6 +230,8 @@ def process_samples(
         if str(name) in {"PauliX", "PauliY", "PauliZ", "Hadamard"}:
             # Process samples for observables with eigenvalues {1, -1}
             samples = 1 - 2 * qml.math.squeeze(samples)
+            if samples.shape == ():
+                samples = samples.reshape((1,))
```

Comment:
I'm surprised this doesn't break anything else 🤔

Reply:
I guess we don't really test 1 shot much...